Build issue Qt 5.12.6, qset

**Describe the bug**
During build from source, step `make -j$(nproc)` fails due to unexpected function call.

**To Reproduce**
Steps to reproduce the behavior:
1. Follow build for Linux steps in README.md
2. Observe output on step which asks to run command -j$(nproc)

**Expected behavior**
Build to succeed.

**Desktop (please complete the following information):**
 - OS: Xubuntu
 - Version 18.04

**Additional context**
filemanager.cpp: In member function ‘void FileManager::loadDeployemendFiles(const QString&)’:
filemanager.cpp:55:82: error: no matching function for call to ‘QSet<QString>::QSet(QList<QString>::iterator, QList<QString>::iterator)’
 deployedFiles.unite(QSet<QString>(deployedFiles.begin(), deployedFiles.end()));
                                                                             ^
In file included from /opt/Qt/5.12.6/gcc_64/include/QtCore/QSet:1:0,
                 from filemanager.h:11,
                 from filemanager.cpp:10:
/opt/Qt/5.12.6/gcc_64/include/QtCore/qset.h:61:12: note: candidate: QSet<T>::QSet(std::initializer_list<_Tp>) [with T = QString]
     inline QSet(std::initializer_list<T> list)
            ^~~~
/opt/Qt/5.12.6/gcc_64/include/QtCore/qset.h:61:12: note:   candidate expects 1 argument, 2 provided
/opt/Qt/5.12.6/gcc_64/include/QtCore/qset.h:59:12: note: candidate: QSet<T>::QSet() [with T = QString]
     inline QSet() Q_DECL_NOTHROW {}
            ^~~~
/opt/Qt/5.12.6/gcc_64/include/QtCore/qset.h:59:12: note:   candidate expects 0 arguments, 2 provided
/opt/Qt/5.12.6/gcc_64/include/QtCore/qset.h:54:7: note: candidate: QSet<QString>::QSet(const QSet<QString>&)
 class QSet
       ^~~~
/opt/Qt/5.12.6/gcc_64/include/QtCore/qset.h:54:7: note:   candidate expects 1 argument, 2 provided
/opt/Qt/5.12.6/gcc_64/include/QtCore/qset.h:54:7: note: candidate: QSet<QString>::QSet(QSet<QString>&&)
/opt/Qt/5.12.6/gcc_64/include/QtCore/qset.h:54:7: note:   candidate expects 1 argument, 2 provided
Makefile:2143: recipe for target 'filemanager.o' failed
make[1]: *** [filemanager.o] Error 1
make[1]: *** Waiting for unfinished jobs....

--- a/CQtDeployer.pro
+++ b/CQtDeployer.pro
@@ -7,6 +7,12 @@
 
 TEMPLATE = subdirs
 CONFIG += ordered
+
+lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 14) {
+    message(Tests is disabled!)
+    DEFINES += WITHOUT_TESTS
+}
+
 !android {
     SUBDIRS += QuasarAppLib \
                Pe \

--- a/Deploy/filemanager.cpp
+++ b/Deploy/filemanager.cpp
@@ -52,7 +52,11 @@ void FileManager::loadDeployemendFiles(const QString &targetDir) {
 
     QStringList deployedFiles = settings->getValue(targetDir, "").toStringList();
 
+#if QT_VERSION < QT_VERSION_CHECK(5, 14, 0)
+    _deployedFiles.unite(deployedFiles.toSet());
+#else
     _deployedFiles.unite(QSet<QString>(deployedFiles.begin(), deployedFiles.end()));
+#endif
 }
 
 

